src/spirc.rs

```diff
@@ -363,6 +411,45 @@ impl SpircTask {
                 self.notify(None);
             }
 
+            MessageType::kMessageTypeVolumeUp => {
+                let old_vol = self.mixer.volume();
+
+                // Convert the absolute value into a relative value. Than add 10%, check
+                // for boundaries and convert the new relative volume back to an absolute value.
```

Comment:
Why does need to be converted to float ? You can just add 6554 as an integer (there's a small rounding error, but that's fine IMO)